hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaAction.java

```diff
@@ -184,13 +185,15 @@ public String toString() {
     @Override
     public void fireCreateEvent(final DescriptorEvent descriptorEvent) {
         EventPublisherHolder.getInstance().getEventPublisher()
-                .publishEvent(new ActionCreatedEvent(this, EventPublisherHolder.getInstance().getApplicationId()));
+                .publishEvent(new ActionCreatedEvent(this, BaseEntity.getIdOrNull(rollout),
```

Comment:
The `BaseEntity.getIdOrNull` is only use in this class, or? If yes then create a private method, it's cleaner then a static method.